app/abilities/ability.rb

```diff
@@ -16,7 +16,7 @@ def initialize(user)
 
     if Settings.status != "database_offline"
       can [:index, :feed, :show], Changeset
-      can :index, ChangesetComment
+      can :show, :changeset_comments_feed
```

Comment:
As before, I don't think this is the correct action. This isn't a single object, it's a list of objects, so the method should be `index` to reflect this.

Reply:
You could have easily argued for the opposite thing, which I'm not going to do because I just want to move feeds out of the way of #4248.